python-3.10.yaml

```diff
@@ -0,0 +1,108 @@
+package:
+  name: python-3.10
+  version: 3.10.9
+  epoch: 0
+  description: "the Python programming language"
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: PSF-2.0
+  dependencies:
```

Comment:
Should this also `provides: python=3.10.999` ?

(I have the same question about the various Go's and Ruby's etc)